src/libraries/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs

```diff
@@ -73,6 +73,9 @@ internal static int[] GetCurrencyNegativePatterns(string localeName)
 
                 case "fr-CA":
                     return PlatformDetection.IsNlsGlobalization ? new int[] { 15 } : new int[] { 8, 15 };
+
+                case "luy-KE":
```

Comment:
ditto, please avoid adding more hardcoded culture names here. Globalization data changes and this can make the test fail in the future.